File: app/src/main/java/com/example/acplite/entidades/Arbol.java
package com.example.acplite.entidades;

import android.graphics.Bitmap;

public class Arbol {
    private String treeName;
    private String treeScientificName;
    private String treeDescription;
    private Bitmap treeImg;

    public Arbol() {
    }

    public Arbol(String treeName, String treeScientificName, String treeDescription, Bitmap treeImg) {
        this.treeName = treeName;
        this.treeScientificName = treeScientificName;
        this.treeDescription = treeDescription;
        this.treeImg = treeImg;
    }

    public Arbol(String treeScientificName, String treeDescription, Bitmap treeImg) {
        this.treeScientificName = treeScientificName;
        this.treeDescription = treeDescription;
        this.treeImg = treeImg;
    }

    public String getTreeName() {
        return treeName;
    }

    public void setTreeName(String treeName) {
        this.treeName = treeName;
    }

    public String getTreeScientificName() {
        return treeScientificName;
    }

    public void setTreeScientificName(String treeScientificName) {
        this.treeScientificName = treeScientificName;
    }

    public String getTreeDescription() {
        return treeDescription;
    }

    public void setTreeDescription(String treeDescription) {
        this.treeDescription = treeDescription;
    }

    public Bitmap getTreeImg() {
        return treeImg;
    }

    public void setTreeImg(Bitmap treeImg) {
        this.treeImg = treeImg;
    }
}

File: app/src/main/java/com/example/acplite/entidades/Arbol.kt
package com.example.acplite.entidades

import android.graphics.Bitmap

class Arbol {
    var treeName: String? = null
    var treeScientificName: String? = null
    var treeDescription: String? = null
    var treeImg: Bitmap? = null

    constructor() {}
    constructor(
        treeName: String?,
        treeScientificName: String?,
        treeDescription: String?,
        treeImg: Bitmap?
    ) {
        this.treeName = treeName
        this.treeScientificName = treeScientificName
        this.treeDescription = treeDescription
        this.treeImg = treeImg
    }

    constructor(treeScientificName: String?, treeDescription: String?, treeImg: Bitmap?) {
        this.treeScientificName = treeScientificName
        this.treeDescription = treeDescription
        this.treeImg = treeImg
    }
}